ui/src/FeedbackButton.tsx

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import FeedbackIcon from "@material-ui/icons/FeedbackOutlined";
+import Button from "@material-ui/core/Button";
+import { makeStyles } from "@material-ui/core/styles"
+import { ScholarReaderContext, PaperId } from "./state";
+import queryString from "querystring";
+
+function mkFeedbackLink(paperId?: PaperId, extraContext?: Object) {
+    console.log(extraContext);
+    const baseUrl = "https://docs.google.com/forms/d/e/1FAIpQLSdnTn4ng-3SsNqwr6M7yF54IhABNAw9_KIjPdWC746fIe546w/viewform";
+    const params = {
+        // This ID is produced by Google Forms. If the form changes we
+        // may need to change this value
```

Comment:
Does this mean if edits are made to the form the value might change, or if we start using another form?

Reply:
Ah, good question.

The ID is auto-generated by Google, and accordingly the mechanism behind it is opaque to us.  So it could change from any action in theory.

I'll update the docs to clarify.

Reply:
Updated!